### PR TITLE
Report overall and transfer timings as part of hot reload statistics,…

### PR DESCRIPTION
**Pull request by aam**
… analytics.

### CODE REVIEW

**Comment by aam** [comment]
cc @jayoung-lee

**Review by DanTup — APPROVED** [review]
LGTM - cd12 and cd13 will need to be available and configured in the Analytics account again.

**Comment by jayoung-lee** [comment]
Custom dimensions 12 and 13 are added to the Analytics as well.

File: packages/flutter_tools/lib/src/run_hot.dart [hunk 1]
```diff
@@ -770,17 +770,22 @@ class HotRunner extends ResidentRunner {
...
 
     reloadTimer.stop();
-    printTrace('Hot reload performed in ${getElapsedAsMilliseconds(reloadTimer.elapsed)}.');
+    final Duration reloadDuration = reloadTimer.elapsed;
+    final int reloadInMs = reloadDuration.inMilliseconds;
+
+    analyticsParameters[kEventReloadOverallTimeInMs] = '$reloadInMs';
+    flutterUsage.sendEvent('hot', 'reload', parameters: analyticsParameters);
+
+    printTrace('Hot reload performed in $reloadInMs.');
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/run_hot.dart]: this looks like a ui regression? we used to use getElapsedAsMilliseconds() which gives units and suchlike.

(i'll fix it in my timeout patch)

Comment (aam) [reply packages/flutter_tools/lib/src/run_hot.dart]: Oh, thanks. I didn't realize `getElapsedAsMilliseconds()` produces formatted string.

**Pull request merged**